cmd/yurthub/app/config/config.go

```diff
@@ -158,6 +158,10 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 }
 
 func parseRemoteServers(serverAddr string) ([]*url.URL, error) {
+	if serverAddr == "" {
+		return make([]*url.URL, 0)
```

Comment:
how about change to `return []*url.URL{}, nil`?

Reply:
one return value(`error`) is missed.

Reply:
Thinks！ I've added one return value(error).